### PR TITLE
[FIX] Fixes bottom sheets not being scrollable and completely visible

### LINKED ISSUE #2253: [BUG] Chat sorting options not visible in pop-up when phone orientation is landscape.
## Describe the bug

Chat sorting options not visible in pop-up when phone orientation is landscape.

## To Reproduce

1. Hold phone in horizontal orientation.
2. Tap the sorting button at the top right.
3. Sorting options are inaccessible.

## Expected behavior

Sorting options should be accessible in horizontal orientation.

## Devices and Versions

Your Rocket.Chat.Android version: 3.4.0

## GIF

issue horizontal-2253

### PR DESCRIPTION
**Pull request by hfeky**
@RocketChat/android

Closes #2253
Closes #2277

#### Changes:

1. Fixed bottom sheets not being scrollable when the sheet is too long for the device screen. Initially one could not scroll.
2. Fixed bottom sheets not being completely visible when the sheet is too long for the device screen.
3. Fixed hardcoded string _"+99"_ in `ChatRoomFragment`.
4. Corrected 2 translations in Arabic language.

#### GIFs for the change:

**Portrait:**
Bottom Sheet Portrait

**Landscape:**
Bottom Sheet Landscape

### CODE REVIEW

**Comment by philipbrito** [comment]
@HusseinElFeky Are you using the `3.4.1` app version and facing the blank chat list issue?

**Comment by hfeky** [comment]
@filipedelimabrito Unfortunately yes, tried 3.4.1 on Play Store and 3.4.1 (latest commit) from here, on open.rocket.chat server, and no chats are showing. However there is a sneaky **temporary** workaround that I do to be able to chat with users. This only works for users, not channels. If you go to Directory, search for any user, and click on him, a toast will show, saying _"Expected END_ARRAY but was END_OBJECT at path $.update[8].lastMessage"_. It will not open the chat, but if you return to the main chats list, the user you clicked on earlier will appear, and now you can chat with him. I can send you a GIF, showing that if you want. Also if by any way you need help with testing, I can help you.

**Update:** I have sent you a GIF, demonstrating that issue in a DM on Rocket.Chat.

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -103,10 +103,10 @@
     <string name="msg_share_using">نشر بواسطة</string>
     <string name="msg_profile_updated_successfully">تم تحديث الملف الشخصي بنجاح</string>
     <string name="msg_username">اسم المستخدم</string>
-    <string name="msg_username_or_email">اسم المستخدم أو عنوان بريد</string>
+    <string name="msg_username_or_email">اسم المستخدم أو عنوان البريد</string>
```
Comment (philipbrito) [inline app/src/main/res/values-ar/strings.xml:106]: Thanks for improving the translations here :)

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 2]
```diff
@@ -536,7 +536,7 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
                 if (newMessageCount <= 99) {
                     text_count.text = newMessageCount.toString()
                 } else {
-                    text_count.text = "99+"
+                    text_count.text = getString(R.string.msg_more_than_ninety_nine_unread_messages)
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt:539]: Thank you :)

**Review by philipbrito — APPROVED** [review]
Thank you @HusseinElFeky!
👏 👏

**Pull request merged**